jib-gradle-plugin/build.gradle

```diff
@@ -47,17 +47,20 @@ release {
 pluginBundle {
   website = 'https://github.com/GoogleContainerTools/jib/'
   vcsUrl = 'https://github.com/GoogleContainerTools/jib/'
+  tags = ['google', 'java', 'containers', 'docker', 'kubernetes', 'microservices']
+}
 
+gradlePlugin {
   plugins {
     jibPlugin {
       id = 'com.google.cloud.tools.jib'
       displayName = 'Jib'
       description = 'Containerize your Java application'
-      tags = ['google', 'java', 'containers', 'docker', 'kubernetes', 'microservices']
+      implementationClass = 'com.google.cloud.tools.jib.gradle.JibPlugin'
     }
   }
 }
-tasks.publishPlugins.dependsOn integrationTest
+//tasks.publishPlugins.dependsOn integrationTest
```

Comment:
Should we keep this?

Reply:
oh yeah oops, I just needed it to confirm something.